pkg/secrets/mock.go

```diff
@@ -76,6 +77,13 @@ func (me *MockEncrypter) EncryptValue(ctx context.Context, plaintext string) (st
 	return "", errors.New("mock value not provided")
 }
 
+func (me *MockEncrypter) BatchEncrypt(ctx context.Context, secrets []string) ([]string, error) {
+	if me.BatchEncryptF == nil {
```

Comment:
Just for consistency can you do this the same order as EncryptValue. `if != nil then call else error"